R/linux_clipboard.R

```diff
@@ -27,6 +27,12 @@ has_xclip <- function() has_util(c("xclip", "-o", "-selection", "clipboard"))
 # Determine if system has 'xsel' installed
 has_xsel <- function() has_util(c("xsel", "--clipboard"))
 
+# Determine if system has 'wl-paste' installed
+has_wlpaste <- function() has_util(c("wl-paste", "--primary"))
```

Comment:
Please set up a test in `tests/testthat/test_systems.R` to check these functions work.

First, go to `travis.yml` and add an entry in the build matrix that checks for `wl-clipboard`. Follow the pattern that you see for `xsel` and `xclip`

Reply:
You'll also need to add tests ito `tests/testthat/test-diagnostics.R`, again following the pattern you'll see for `xlicp` and `xsel`

Reply:
Can you please name it `has_wl_paste`? (and `has_wl_copy` below)

I don't quite like it when people spell it as `wlcopy`/`wlpaste` is all.